src/etl/arbo/steps/clean-field-names-and-remove-unused-fields-step.ts

```diff
@@ -82,7 +85,7 @@ export const cleanFieldNamesAndRemoveUnusedFieldsStep = (
       seroprevalenceStudy95CIUpper: estimate["Seroprevalence 95% CI Upper"],
       seroprevalenceCalculated95CILower: estimate["Seroprevalence 95% CI Lower (formula)"],
       seroprevalenceCalculated95CIUpper: estimate["Seroprevalence 95% CI Upper (formula)"],
-      country: estimate["Country"],
+      country: estimate["Country archive"],
```

Comment:
Something happened to the "Country" field since I last ran the ETL. I think they moved it to an external table because now I'm getting an ID. Probably a good idea for us to start using that external table in the same way we use the source sheet table at some point, there's some additional fields which would be kind of cool.